wamr-compiler/CMakeLists.txt

```diff
@@ -161,6 +161,13 @@ endif()
 # Enable LLVM
 if (NOT WAMR_BUILD_WITH_CUSTOM_LLVM)
   set (LLVM_SRC_ROOT "${PROJECT_SOURCE_DIR}/../core/deps/llvm")
+
+  # In Debug mode, force using Release runtime library on Windows platform
+  if (WAMR_BUILD_PLATFORM STREQUAL "windows")
+    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} /MD")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MD")
+  endif()
+
```

Comment:
- Does iwasm+llvmjit have the same issue?
- Both pre-built libraries and libraries compiled from source share the same problem; we might want to add `/MD` for both `WAMR_BUILD_WITH_CUSTOM_LLVM` and `not WAMR_BUILD_WITH_CUSTOM_LLVM` scenarios.
- I personally suggest using `add_compile_options()` to achieve greater flexibility and compatibility.